eZ/Publish/Core/Repository/User/PasswordHashServiceInterface.php

```diff
@@ -8,13 +8,14 @@
 
 namespace eZ\Publish\Core\Repository\User;
 
-/**
- * @internal
- */
 interface PasswordHashServiceInterface
 {
     public function getDefaultHashType(): int;
 
+    public function getSupportedHashTypes(): array;
```

Comment:
Since other methods specifically ask for `int`.

```suggestion
    /**
     * @return int[]
     */
    public function getSupportedHashTypes(): array;
```